### PR TITLE
Replace TSlint with ESLint TS plugin.

### PR DESCRIPTION
**Pull request by kibertoad**
Unify Prettier configuration.
Add linting Lerna job.

### CODE REVIEW

File: app/src/interfaces/many-to-many/use-selection.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default function useSelection(
 		const selection = newSelection.reduce((acc, item) => {
 			if (selectedPrimaryKeys.value.includes(item) === false) acc.push({ [junctionField]: item });
 			return acc;
-		}, new Array());
+		}, [] as Record<string, any>[]);
```
Comment (kibertoad) [inline app/src/interfaces/many-to-many/use-selection.ts:50]: This change was necessary, TS was complaining about type of array being `never[]` which does not accept `item`.

File: api/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+module.exports = {
```
Comment (rijkvanzanten) [inline api/.eslintrc.js:1]: Should we do the same Prettier trick for ESLint? I'd like the sub-packages (under `/packages`) and the app to adhere to the same linter rules 🙂

File: api/.eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	root: true,
+	env: {
+		node: true,
+	},
+	extends: [
+		'plugin:@typescript-eslint/recommended',
+		// 'plugin:prettier/recommended'
+		// Uncomment when we are ready to prettify everything
```
Comment (rijkvanzanten) [inline api/.eslintrc.js:9]: Lets wrap up this PR with the configs, and then do 1 more PR auto-fixing everything based on the new rules 👍

File: api/.eslintrc.js (hunk 2 above)
Comment (kibertoad) [reply api/.eslintrc.js:3]: Done!

**Review by rijkvanzanten — DISMISSED** [review]

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**